src/Compilers/Core/Portable/SourceGeneration/Nodes/SyntaxValueProvider_ForAttributeWithMetadataName.cs

```diff
@@ -4,20 +4,14 @@
 
```

Comment:
SyntaxValueProvider_ForAttributeWithSimpleName should be reviewed before this file. it will make this file easier to understand.